packages/react-component-library/src/components/DismissibleBanner/partials/StyledDescription.tsx

```diff
@@ -0,0 +1,10 @@
+import { selectors } from '@royalnavy/design-tokens'
+import styled from 'styled-components'
+
+const { color, fontSize } = selectors
+
+export const StyledDescription = styled.p`
+  color: ${color("neutral", "400")};
+  font-size: ${fontSize("base")};
```

Comment:
I think these should be single quotes `''`. 

Prettier usually picks that up on save, I think.